qsirecon/interfaces/dipy.py

```diff
@@ -190,33 +231,182 @@ def _extrapolate_scheme(self, scheme_name, runtime, fit_obj, mask_img):
 
 
 class MAPMRIInputSpec(DipyReconInputSpec):
-    radial_order = traits.Int(6, usedefault=True)
-    laplacian_regularization = traits.Bool(True, usedefault=True)
-    laplacian_weighting = traits.Either(traits.Str("GCV"), traits.Float(0.2), usedefault=True)
-    positivity_constraint = traits.Bool(False, usedefault=True)
-    pos_grid = traits.Int(15, usedefault=True)
+    radial_order = traits.Int(
+        6,
+        usedefault=True,
+        desc="An even integer that represents the order of the basis.",
+        doc=ConditionalDoc("The radial order of the MAPMRI model was set to {value}."),
+        recon_spec_accessible=True,
+    )
+    laplacian_regularization = traits.Bool(
+        True,
+        usedefault=True,
+        desc="Regularize using the Laplacian of the MAP-MRI basis.",
+        doc=ConditionalDoc(
+            "The Laplacian of the MAP-MRI basis was used for regularization.",
+            if_false="The Laplacian of the MAP-MRI basis was not used for regularization.",
+        ),
+        recon_spec_accessible=True,
+    )
+    laplacian_weighting = traits.Either(
+        "GCV",
+        traits.List(traits.Float()),
+        traits.Float(0.2),
+        default="GCV",
+        usedefault=True,
+        desc=(
+            "The string 'GCV' makes it use generalized cross-validation to find "
+            "the regularization weight. A scalar sets the regularization weight "
+            "to that value and an array will make it selected the optimal weight "
+            "from the values in the array."
+        ),
+        doc=ConditionalDoc("Laplacian weighting was set to {value}."),
```

Comment:
this is an instance where the ConditionalDoc is maybe a little underpowered. The behavior can be pretty different depending on what this is set to. Although the DIPY docs describe it nicely

Reply:
I noticed that exact issue. Plus the _references_ should differ based on the value. We might want to figure out a way to support specific values in the ConditionalDoc, beyond True, False, and Undefined.